### ISSUE TITLE
无法在1.13版本中使用

### ISSUE THREAD

**Issue by NyanMeow** [connection-lost/LockettePro#88]
由于1.13的ID变更+数字ID的废除导致本插件无法正常锁箱子
即使在config里面的lockables写上了对应ID也无法使用

**Comment by Skajdrowski** [comment]
Yeah We need the 1.13 update. Spigot got released 1.13 version while ago

**Comment by Thatsmusic99** [comment]
Sad truth is, looking at the dev's recent activity, I can't say one will be guaranteed but I'm hoping so. 1.13 is a tricky update for devs to update to, not to mention make compatible with older versions. In the meantime though, I may as well fork my own version and see if a pull request for this can be made. Judging that the plugin is said to use up to date methods, it shouldn't be that bad to deal with. ;P

**Comment by Skajdrowski** [comment]
If your fork will be done, it will pop up on your profile ?

**Comment by Thatsmusic99** [comment]
It does come up in my profile (repos) but no changes have been pushed. Plus I'm already experiencing random issues with signs not rotating properly. Plus I want to ensure it's still backwards compatible by the time it's done (as hard as it is, but I've already done it once...)

**Comment by Ghost-chu** [comment]
@NyanMeow @Skajdrowski @Thatsmusic99 
I make clone and build a 1.13 support LockettePro from NyaaCat repo:https://pan.baidu.com/s/1nIU93vc_oHqatV2njL27PQ
If you use WorldGuard string-id version, you will need SunnySide Community fix edition:https://pan.baidu.com/s/1mdjQM3S8E0OYa1NdaEGa6w

NyaaCat repo:https://github.com/NyaaCat/LockettePro

**Comment by Skajdrowski** [comment]
@Ghost-chu Is this 100% stable and performancefully fork ?

**Comment by Ghost-chu** [comment]
Yes @skajdrowski

**Comment by NyanMeow** [comment]
是鬼畜dalao
NyaaCat的魔改版正好在测试用
原作者估计是弃坑了

NyaaCat's reset version is just in test
The original author estimated that it was abandoned.

**Comment by Skajdrowski** [comment]
@Ghost-chu Working well. Thank you

**Comment by SeanWalk** [comment]
any other link can download nyaacat version? the link is dead

**Comment by Thatsmusic99** [comment]
Just came back from not having access to my laptop for a week so I ended up having to stop, but thank you @Ghost-chu , you've just saved me a lot of pain trying to update all the plugins we use that are no longer developed.

And @SeanWalk the download link works fine for me, did you try the https://pan.baidu.com/s/1nIU93vc_oHqatV2njL27PQ link?

**Comment by SeanWalk** [comment]
@Thatsmusic99 the link is 
啊哦，你所访问的页面不存在了。
I can't download it

**Comment by Skajdrowski** [comment]
@SeanWalk https://mega.nz/#!7FxEBKCQ!x0cmdpmfV7WkX2VfViKN0JlnRta_ADji-GtUf_S2GnM

**Comment by Skajdrowski** [comment]
@Ghost-chu Do you have the 1.13.1 version ?

**Comment by cyilin** [comment]
@Skajdrowski https://github.com/NyaaCat/LockettePro/releases

**Comment by Skajdrowski** [comment]
@cyilin Thank you
